FreeRTOS-Plus/Demo/AWS/Fleet_Provisioning_Windows_Simulator/Fleet_Provisioning_With_CSR_Demo/demo_config.h

```diff
@@ -145,6 +145,10 @@ extern void vLoggingPrintf( const char * pcFormatString,
  * @note The provisioning template MUST be created in AWS IoT before running the
  * demo.
  *
+ * @note If you followed the manual setup steps on https://freertos.org/iot-fleet-provisioning/demo.html,
+ * the provisioning template name is "FleetProvisioningDemoTemplate".
+ * However, if you used CloudFormation to set up the demo, the template name is "CF_FleetProvisioningDemoTemplate"
```

Comment:
Why is the name different?

Reply:
Users may want to work through setup steps manually in addition to using CloudFormation. However, if any created resources share names with preexisting resources, then the CloudFormation flow will fail. 

Separating these out allows users to tweak the setup flow manually without breaking CloudFormation, if they wish to use it later.